### PR TITLE
CylcCon 2020 workshop report draft.

### PR DESCRIPTION
**Pull request by hjoliver**
What it says.  Reviewers please check:
- have I missed any important items?
- are the tentative completion dates reasonable?

I need to make this available pretty quickly if possible.

### CODE REVIEW

File: docs/feb2020-workshop-report.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+The following are tentative endpoints to aim for, for the major tasks. (Some
+smaller tasks are not listed here as they can be fitted in around the bigger
+ones - see the list above for them). Many of these items are already in
+progress or have at least been planned in advance. Initials added for those
+leading the development, but others may also be involved, and we have to be
+flexible depending on availability.
+
+- March 2020:
+  - spawn-on-demand decision (HO, DM, OS)
```
Comment (oliver-sanders) [inline docs/feb2020-workshop-report.md]: This one might be a little optimistic, I'm not sure.

File: docs/feb2020-workshop-report.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  - complete spawn-on-demand, if agreed, e.g. adapt the test battery (HO)
+- May 2020:
+  - UI integrated control functionality (BK, OS)
+  - Complete remaining UI design elements - xtriggers etc. (BK, OS)
+- June 2020:
+  - platforms support and config changes (TP, MH)
+  - authorization at the UI Server (HO)
+  - thoroughly document security (JR)
+- July 2020:
+  - `rose suite-run` migration (TP, OS)
```
Comment (oliver-sanders) [inline docs/feb2020-workshop-report.md]: July is a bit optimistic for this one.

File: docs/feb2020-workshop-report.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  - UI Server access to historical data within the view window (DS)
+  - cross-user functionality (BK, OS)
+- August 2020:
+  - new run command semantics and run directory structure (HO)
+  - `cylc review` decision - integration or contingency? (OS)
+  - review and upgrade all documentation and training material (ALL)
```
Comment (oliver-sanders) [inline docs/feb2020-workshop-report.md]: Pretty big job, August may be optimistic.

**Comment by oliver-sanders** [comment]
Fine but timeframes are scary.

**Comment by hjoliver** [comment]
> Fine but timeframes are scary.

True. I'll push those you've highlight back a bit.  I was trying to aim for the September testing release "deadline" that @dpmatthews thought was very important.

**Comment by hjoliver** [comment]
Feedback addressed. If you approve, we can merge this and make it official once @dpmatthews has taken a look.

**Comment by hjoliver** [comment]
@dpmatthews - I need to merge this so I can link to the "published" version (on GitHub) in a funding proposal today.  I will not proclaim it to be "official" or alert others to it until you have approved it however. Hope that's OK.

File: docs/feb2020-workshop-report.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+- Control capability has been implemented in the web UI, but not integrated
+   into the primary views yet.
+- Back-end authentication (between CLI and workflow schedulers) is mostly done
+
+## Prioritization of remaining tasks
+
+(First read the [Executive summary](#executive-summary).)
+
+The following items were thoroughly discussed, and problems worked out, in the
+workshop. Now they just need to implemented or completed.
```
Comment (dpmatthews) [inline docs/feb2020-workshop-report.md:106]: ... to be ...

File: docs/feb2020-workshop-report.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+- June 2020:
+  - platforms support and config changes (TP, MH)
+  - authorization at the UI Server (HO)
+  - thoroughly document security (JR)
+- July 2020:
+  - UI Server access to historical data within the view window (DS)
+  - cross-user functionality (BK, OS)
+- August 2020:
+  - `rose suite-run` migration (TP, OS)
+  - new run command semantics and run directory structure (HO)
```
Comment (dpmatthews) [inline docs/feb2020-workshop-report.md:211]: I think this is part of the rose suite-run migration

File: docs/feb2020-workshop-report.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+A recent screenshot of the new web UI showing live workflow data:
+![Cylc 8 web UI as of February 2020](img/cylc-8-ui-feb2020.png)
+
+As of February 2020:
+- The previously monolithic Cylc project has been split into multiple modular
+  projects on GitHub (primarily because the UI is no longer Python based)
+- Our new communications platforms Riot (developer chat and video
+  conferencing) and Discourse (mail forum) are proving effective
+- Web UI design settled after extensive consultation with users
```
Comment (dpmatthews) [inline docs/feb2020-workshop-report.md:78]: I know we've consulted via Discourse but not all users will have seen this and many won't really understand some of the changes until they get a chance to try it out. I wouldn't want them to think it is fixed in stone. How about something like:
"Web design settled after consultation with users (although we anticipate lots of feedback during user testing)"

**Comment by hjoliver** [comment]
Thanks @dpmatthews, to save time I've made your suggested changes and pushed directly to master without a PR (as they won't be controversial).

**Pull request merged**